bot_code/trainer/configs/randomised_trainer.cfg

```diff
@@ -1,8 +1,8 @@
 [Randomised Trainer Configuration]
 total_batches = 40000
 save_step = 50
-#teacher_package = TutorialBot.tutorial_bot_output
-teacher_package = TutorialBot.atba2_scorer
+#teacher_package =  models.TutorialBot.tutorial_bot_output
+teacher_package =  models.TutorialBot.atba2_scorer
```

Comment:
Did the tutorial bots gets moved inside models?
That might get confusing as they are not a model themselves.

Reply:
Removing this as we have a plan where to move TutorialBot instead. (group it close with FakeModel)

I'll squash these commits once you think we should submit it in this state.